pdvega/_pandas_internals.py

```diff
@@ -1,6 +1,9 @@
 from pandas.plotting._core import FramePlotMethods, SeriesPlotMethods
 
 try:
+    from pandas.api.extensions import (register_dataframe_accessor,
+                                       register_series_accessor)
+except ImportError:
     from pandas.core.accessor import AccessorProperty
 except ImportError:  # Pandas before 0.22.0
     from pandas.core.base import AccessorProperty
```

Comment:
A ``try`` block can't have two identical except statements... this would have to be a nested try-except within the except.

Reply:
Oops! This was supposed to be nested 🤦‍♂️. This is residual code that I forgot to go back and fix. Sorry about that!